certs/certs.go

```diff
@@ -30,3 +39,46 @@ type Repository interface {
 	// RetrieveBySerial retrieves a certificate for a given serial ID
 	RetrieveBySerial(ctx context.Context, ownerID, serialID string) (Cert, error)
 }
+
+func LoadCertificates(caPath, caKeyPath string) (tls.Certificate, *x509.Certificate, error) {
+	var tlsCert tls.Certificate
+	var caCert *x509.Certificate
+
+	if caPath == "" || caKeyPath == "" {
+		return tlsCert, caCert, nil
+	}
+
+	if _, err := os.Stat(caPath); os.IsNotExist(err) {
+		return tlsCert, caCert, err
+	}
+
+	if _, err := os.Stat(caKeyPath); os.IsNotExist(err) {
+		return tlsCert, caCert, err
+	}
```

Comment:
Do we also care about other errors besides `NotExists`  (such as insufficient privilege)?